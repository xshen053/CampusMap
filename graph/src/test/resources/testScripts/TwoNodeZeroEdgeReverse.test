# Tests adding two nodes to the graph

# Create a graph
CreateGraph TNZER_graph

# Add two nodes
AddNode TNZER_graph b
AddNode TNZER_graph a

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
ListNodes TNZER_graph
ListChildren TNZER_graph a
ListChildren TNZER_graph b