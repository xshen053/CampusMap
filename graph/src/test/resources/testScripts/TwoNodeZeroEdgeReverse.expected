# Tests adding two nodes to the graph

# Create a graph
created graph TNZER_graph

# Add two nodes
added node b to TNZER_graph
added node a to TNZER_graph

# Print the nodes in the graph, the outgoing edges from a,
# and outgoing edges from b
TNZER_graph contains: a b
the children of a in TNZER_graph are:
the children of b in TNZER_graph are: